### ISSUE TITLE
gmail detects virus

### ISSUE THREAD

**Issue by Dokazz** [UnamSanctam/SilentCryptoMiner#947]
I builded my silent miner, so its in .exe, 
i tried sending it to my other email adress to see how it looked, but it says it contains a virus.
Therefore i can't spread my silent miner
Did i do something wrong in the process?

**Comment by UnamSanctam** [comment]
Google has it's own virus scanner like VirusTotal, either you will have to use an obfuscator to get it undetected enough (quite difficult since it seems Google scans with many different antiviruses) or you can zip/rar the .exe with a password to possibly get around it.

**Comment by Dokazz** [comment]
okay thank you very much, but do you think some people will open the rar and download what is in there?

**Comment by Dokazz** [comment]
hey i did a .rar, but now it does not keep the icon i chose for the .exe
is there any way to still keep the custom icon i chose in the .rar?
thank you for your quick answers btw

**Comment by UnamSanctam** [comment]
> okay thank you very much, but do you think some people will open the rar and download what is in there?

It depends on what you're sending.

> hey i did a .rar, but now it does not keep the icon i chose for the .exe is there any way to still keep the custom icon i chose in the .rar? thank you for your quick answers btw

If you're using WinRar then it might not show the icons when you open the .rar but only when you extract it.

**Comment by Dokazz** [comment]
Okay thank you for your answers, but now i am facing another problem:
gmail does not want to send .exe's apparently, it does not say it is a virus or anything, but it just blocks the .rar file
is there a way to convert the miner into something else than .exe?

**Comment by UnamSanctam** [comment]
Could be that gmail does not allow encrypted .zip files, you cannot really change the miner into something else, only .exe files can be executed, though there are exploits/methods which allow you to embed it into other things. Nowadays email providers have such good antispam/antivirus systems that generally people don't send malware through email anymore.

**Comment by Dokazz** [comment]
Does the people i send the silent miner to have to execute the file or just download it btw?
i've never been sure about this part of the process

**Comment by UnamSanctam** [comment]
Execute it, there isn't really anything that can run by simply downloading it (since it would be a huge security flaw).

**Comment by Dokazz** [comment]
Okay i understand
Isnt it very hard to spread this silent miner then?

**Comment by UnamSanctam** [comment]
Not really, you would spread it exactly the same as any other programs/malware/miners, there isn't any of them that doesn't have to be executed.

**Comment by Dokazz** [comment]
okay last question before I stop spamming you x)
do you have any tips on how to make people download this .exe and make them execute it?
thank you for your time anyways, very helpful from you

**Comment by UnamSanctam** [comment]
The most common method is spreading it as a fake or real program/game crack or game cheat/hack, there's also different kinds of programs that people need or want, the most common is cracks due to people expecting cracks to already be detected by antiviruses so they don't care about detections and because so many look for them.
